src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,50 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+  }
 }
 
+Animal.alive = [];
+
+Animal.checkIsAlive = () => {
+  for (const animal of Animal.alive) {
+    if (animal.health === 0) {
```

Comment:
what if the task will implement the bite with different damage? let's check if animals have 0 or less health